RealmTasks Apple/RealmTasks iOS/CellPresenter.swift

```diff
@@ -70,26 +72,26 @@ class CellPresenter<Item: Object where Item: CellPresentable> {
 
     private(set) var currentlyEditingCell: TableViewCell<Item>? {
         didSet {
-            viewController.tableView.scrollEnabled = !currentlyEditing
+            viewController.tableView.isScrollEnabled = !currentlyEditing
         }
     }
-    private(set) var currentlyEditingIndexPath: NSIndexPath?
+    private(set) var currentlyEditingIndexPath: IndexPath?
 
     func cellDidBeginEditing(editingCell: TableViewCell<Item>) {
         items.realm?.beginWrite()
         currentlyEditingCell = editingCell
         let tableView = viewController.tableView
 
-        currentlyEditingIndexPath = tableView.indexPathForCell(editingCell)
+        currentlyEditingIndexPath = tableView.indexPath(for: editingCell)
 
-        let editingOffset = editingCell.convertRect(editingCell.bounds, toView: tableView).origin.y - tableView.contentOffset.y - tableView.contentInset.top
+        let editingOffset = editingCell.convert(editingCell.bounds, to: tableView).origin.y - tableView.contentOffset.y - tableView.contentInset.top
         viewController.setTopConstraintTo(constant: -editingOffset)
         tableView.contentInset.bottom += editingOffset
 
-        viewController.setPlaceholderAlpha(0)
+        viewController.setPlaceholderAlpha(alpha: 0)
```

Comment:
method should be refactored to be `setPlaceholderAlpha(0)` at the call site IMO.